utility/BUILD.bazel

```
@@ -7,10 +7,15 @@ load("@dagger//:workspace_defs.bzl", "dagger_rules")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 load("//utility:utility_test.bzl", "utility_test")
 
+MIGRATED_PROD_FILES = glob(["src/main/java/org/oppia/android/util/android/**/*.kt"])
```

We need to exclude the files inside [accessibility](https://github.com/oppia/oppia-android/tree/develop/utility/src/main/java/org/oppia/android/util/accessibility)

I guess you need to add a kdoc here similar to app/BUILD.bazel.

```suggestion
MIGRATED_PROD_FILES = glob(["src/main/java/org/oppia/android/util/accessibility/**/*.kt"])
```

---

Ah. I'm not sure how I missed this--thanks! Will update it locally a bit later and make sure everything's working as expected.

---

Done.